components/twitter/actions/simple-search/simple-search.mjs

```diff
@@ -49,7 +49,7 @@ module.exports = {
     // run paginated search
     return await this.twitter.paginatedSearch({
```

Comment:
Since this is an async function. [You don't need to `return await`](https://eslint.org/docs/rules/no-return-await). Could you please remove it?